workers/unity/Packages/com.improbable.gdk.core/Config/ReceptionistConfig.cs

```diff
@@ -48,5 +48,16 @@ public static ReceptionistConfig CreateConnectionConfigForPhysicalAndroid(string
                 UseExternalIp = true
             };
         }
+
+        public static ReceptionistConfig CreateConnectionConfigForiOS()
+        {
+            return new ReceptionistConfig
+            {
+                // By default this config only connects to local IP (127.0.0.1)
+                // Replace this code with hardcoded IP to coneect to non-local deployment
+                ReceptionistHost = RuntimeConfigNames.ReceptionistHost,
```

Comment:
`RuntimeConfigDefaults` not `RuntimeConfigNames`

Reply:
Oops, wrong copypaste :)